src/Microsoft.VisualStudio.ProjectSystem.Managed/ImmutableCollectionLinqExtensions.cs

```diff
@@ -49,7 +49,7 @@ internal static class ImmutableCollectionLinqExtensions
                     return obj;
             }
 
-            return default;
+            return default!;
```

Comment:
Unfortunately the attribute above doesn't influence the member body, so we need to `!` force this here. Tracked by https://github.com/dotnet/roslyn/issues/36039.